backend/entrypoint.test

```diff
@@ -1,16 +1,15 @@
 #!/bin/bash
-./wait-for rabbitmq:5672 -t 0
-./wait-for postgres:5432 -t 0
+./wait-for ${RABBITMQ_HOST}:${RABBITMQ_PORT} -t 0
+./wait-for ${DATABASE_HOST}:${DATABASE_PORT} -t 0
 
-version=$(PGPASSWORD=$DATABASE_PASSWORD psql -U $DATABASE_USER -d $DATABASE_NAME -h $DATABASE_HOST -c "select version from db_details;" -tA)
+version=$(PGPASSWORD=$DATABASE_PASS psql postgres://${DATABASE_USER}:${DATABASE_PASS}@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_NAME} -c "select version from db_details;" -tA)
```

Comment:
this is fully equivalent to the new version? Just make sure to test.

Reply:
it's the same command